bin/ycsb

```diff
@@ -140,9 +140,9 @@ def is_distribution():
 # presumes maven properly handles system-specific path separators
 def get_classpath_from_maven(database):
     try:
-        debug("Running 'mvn -pl com.yahoo.ycsb:"+database+"-binding -am package -DskipTests "
+        debug("Running 'mvn -pl com.yahoo.ycsb:"+database+" -am package -DskipTests "
               "dependency:build-classpath -DincludeScope=compile -Dmdep.outputFilterFile=true'")
-        mvn_output = subprocess.check_output(["mvn", "-pl", "com.yahoo.ycsb:"+database+"-binding",
+        mvn_output = subprocess.check_output(["mvn", "-pl", "com.yahoo.ycsb:" + database,
```

Comment:
oh I see, we'd pass in 'hbase10-binding' when we call the method now.

could you update get_classpath_from_maven to specify that the parameter is 'module' now rather than 'database'? that would make this much clearer.